src/site/content/en/blog/richer-offline-w-periodic-background-sync/index.md

```diff
@@ -0,0 +1,458 @@
+`'content-sync'` and the `minInterval` is one day.
+
+```js/3-5
+const registration = await navigator.serviceWorker.ready;
+if ('periodicSync' in registration) {
+  try {
+    registration.periodicSync.register('content-sync', {
+      // An interval of one day.
+      minInterval: 24 * 60 * 60 * 1000,
+    });
+  } catch (error) {
+    // Periodic background sync cannot be used.
+  }
+}
+```
+
+### Verifying a registration
+
+Call `periodicSync.getTags()` to retrieve an array of registration tags. The
+example below uses tag names to confirm that cache updating is active to avoid
+updating again.
+
+```js/2,4
+const registration = await navigator.serviceWorker.ready;
+if ('periodicSync' in registration) {
+  const tags = await registration.periodicSync.getTags();
+  // Only update content if sync isn't set up.
+  if (!tags.includes('content-sync')) {
+    updateContentOnPageLoad();
+  }
+} else {
+  // If periodic background sync isn't supported, always update.
+  updateContentOnPageLoad();
+}
+```
+
+You can also use `getTags()` to show a list of active registrations in your web
+app's settings page so that users can enable or disable specific types of
+updates.
+
+### Responding to a periodic background sync event
+
+To respond to a periodic background sync event add a `periodicsync` event
+handler to your service worker. The `event` object passed to it will contain a
+`tag` parameter matching the value used during registration. For example a
+periodic background sync was registered with the name `'content-sync'`, then
+`event.tag` will be `'content-sync'`.
+
+```js
+self.addEventListener('periodicsync', (event) => {
+  if (event.tag === 'content-sync') {
+    // See the "Think before you sync" section for
+    // checks you could perform before syncing.
+    event.waitUntil(syncContent());
+  }
+  // Other logic for different tags as needed.
+});
+```
+
+### Unregistering a sync
+
+To end a registered sync, call `periodicSync.unregister()` with the name of the
+sync you want to unregister.
+
+```js
+const registration = await navigator.serviceWorker.ready;
+if ('periodicSync' in registration) {
+  registration.periodicSync.unregister('content-sync');
+}
+```
+
+## Interfaces
+
+Here's a quick run down of the interfaces provided by the Periodic Background
+Sync API.
+
+<dl>
+  <dt>PeriodicSyncEvent</dt>
+  <dd>Passed to the `ServiceWorkerGlobalScope.onperiodicsync` event handler at a time of the browser's choosing.</dd>
+  <dt>PeriodicSyncManager</dt>
+  <dd>Registers and unregisters periodic syncs and provides tags for registered sync. Retrieve an instance of this class from the ServiceWorkerRegistration.periodicSync` property.</dd>
+  <dt>ServiceWorkerGlobalScope.onperiodicsync</dt>
+  <dd>Registers a handler to receive the `PeriodicSyncEvent`.</dd>
+  <dt>ServiceWorkerRegistration.periodicSync</dt>
+  <dd>Returns a reference to the `PeriodicSyncManager`.</dd>
+</dl>
```

Comment:
This doesn't render as intended: https://deploy-preview-1835--web-dev-staging.netlify.com/richer-offline-w-periodic-background-sync/

![Screen Shot 2019-11-06 at 10 30 03](https://user-images.githubusercontent.com/145676/68285898-69db1200-0080-11ea-8eb6-1d05d0024df0.png)

Reply:
It's not currently accounted for in the stylesheet. @robdodson Do you want me to add formatting to the <dt> elements or will you have an update merged soon?